docs/security/authentication/active-directory/index.md

```diff
@@ -44,21 +44,96 @@ Octopus is built on top of HTTP.sys, the same kernel driver that IIS is built on
 When the link is clicked, it redirects to a page which is configured to tell HTTP.sys to issue the browser challenge. The browser and HTTP.sys negotiate the authentication just like an IIS website would. The user principal is then passed to Octopus. Octopus will then query Active Directory for other information about the user.
 :::
 
-### Using Negotiate to take advantage of Kerberos {#ActiveDirectoryAuthentication-UsingNegotiate}
 
-By default `IntegratedWindowsAuthentication` & `Negotiate` will try to use `Kerberos` first, and then fall back to `NTLM` if it fails.
+### **Kerberos vs NTLM security for AD Authentication** {#ActiveDirectoryAuthentication-NTLMvKerberos}
 
-:::hint
-**Server configuration required**
-Please note that this may require additional configuration to work correctly in your network. Generally, this will require a Service Principal Name (SPN) to be set for the `HTTP` service class, using the machine name. For example, if you have an Octopus Deploy hosted on the `od.mydomain.local` domain name, that is installed on a machine called `winserv2019` then the corresponding SPN will be needed:
-:::
+It is possible to use either `NTLM` or `Kerberos` authentication for your Active Directory authentication. By default selecting `IntegratedWindowsAuthentication` & `Negotiate` will result in Octopus Deploy attempting to use `Kerberos` first, and then silently falling back to `NTLM` if `Kerberos` authentication fails. 
+
+Without some additional configuration, AD authentication, whether forms based or integrated, would usually fail on `kerberos` authentication and failback to `NTLM`.
+
+### **Configuring Kerberos Authentication for Active Directory** {#ActiveDirectoryAuthentication-ConfiguringKerberos}
+
+The following configuration is required for Kerberos authentication
+- A valid Service Principal Name (SPN) for the `HTTP` service class for each Octopus host NETBIOS name. If you are accessing your Host via it's FQDN then you will need to also add a FQDN also for the `HTTP` service class. (Please Note: Whether you've configured your Octopus host to use `HTTP` or `HTTPS`, you will only need to set a `HTTP` SPN.)
```

Comment:
Changed `a`s to `an`s for words that start with a vowel sound (though not actually a vowel).
```suggestion
- A valid Service Principal Name (SPN) for the `HTTP` service class for each Octopus host NETBIOS name. If you are accessing your Host via its FQDN then you will need to also add an FQDN also for the `HTTP` service class. (Please Note: Whether you've configured your Octopus host to use `HTTP` or `HTTPS`, you will only need to set an `HTTP` SPN.)
```